packages/core/src/components/modal/modal.tsx

```diff
@@ -123,7 +123,10 @@ export class TdsModal {
   handleClose = (event) => {
     const closeEvent = this.tdsClose.emit(event);
     if (!closeEvent.defaultPrevented) {
-      this.isShown = false;
+      // respecting passed show prop
+      if (this.show === undefined) {
+        this.isShown = false;
+      }
```

Comment:
I think this makes sense as this is what we say for the show prop:
```
  /** Controls whether the Modal is shown or not. If this is set hiding and showing
   * will be decided by this prop and will need to be controlled from the outside. */
  @Prop() show: boolean;
```

My concern is should we then block the` initializeModal()` method too? In a way, both opening and closing then would be fully under control of a user by toggling state of show prop.